app/components/ui/Modals/AdminPanel/components/FormatDescriptionForm.tsx

```diff
@@ -69,7 +69,7 @@ export function FormatDescriptionForm({ id, subTitle }: FormatDescriptionFormPro
   }, [isEditing])
 
   return (
-    <h1 className="flex flex-row">
+    <h1 className="flex flex-row justify-center tablet:justify-start mt-4 tablet:mt-0">
```

Comment:
its obviouly - next time add descriotion like 'for tablets its centered - for tablet+ decives its justify start'